### PR TITLE
New design for signup page in the Membership flow

### PR DESCRIPTION
**Pull request by svillafe**
## Description

A new Sign Up page for membership is added. Currently, the new design is turned off by default. If you want to see it pass as a clientId=membersB.

### Mobile

screencapture-profile-thegulocal-register-1479290168939

### Desktop

screencapture-profile-thegulocal-register-1479290139451

### CODE REVIEW

File: public/components/membership/_membership_a.css [hunk 1]
```diff
@@ -14,7 +14,6 @@
       background-position: -1010px top;
     }
   }
-
```
Comment (SiAdcock) [inline public/components/membership/_membership_a.css]: Let's kill this unnecessary change 😄

File: public/_layout.css [hunk 2]
```diff
@@ -69,6 +69,47 @@ body > section:last-of-type {
   }
 }
 
+%page-block-v2 {
```
Comment (SiAdcock) [inline public/_layout.css]: Could we be more descriptive with these names? `v2`, `v3` don't convey much information

File: public/components/membership/_membership_b.css [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+  }
+
+  .header {
+    background-image: inline("bg-masthead-72px.png");
+    background-position: center top;
+
+    @media (--viewport-max-tablet) {
+      background-position: -1010px top;
+    }
+  }
```
Comment (SiAdcock) [inline public/components/membership/_membership_b.css]: Styles for `.header__logo-link` and `.header` are identical. Can we refactor to remove duplication?

File: public/components/membership/_membership_b.css [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+    text-align: right;
+
+    p {
+      margin: 0;
+      color: #767676;
+    }
+  }
+
+  .register__section {
+    background-color: white;
```
Comment (SiAdcock) [inline public/components/membership/_membership_b.css]: Hex it up 😄

File: public/components/progress-bar-mem/chevron-01.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="10" height="28" viewBox="0 0 10 28"><path fill="#F6F6F6" d="M0 0h10v28H0z"/><path fill="#AAD8F1" d="M0 0l9 14-9 14"/></svg>
```
Comment (SiAdcock) [inline public/components/progress-bar-mem/chevron-01.svg]: I wonder if we can kill one of these paths? We only need the path with the arrow part, the other path is just the background colour, right?

File: public/components/membership/_membership_a.css [hunk 6]
```diff
@@ -1,4 +1,5 @@
-.skin-members {
+.skin-members,
+.skin-membersB {
```
Comment (SiAdcock) [inline public/components/membership/_membership_a.css]: This file is no longer specific to the `membership_a` variant. Do you think we should we merge `membership_a.css` and `membership_b.css` into one file?

File: public/components/membership/_membership_b.css [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+  float: left;
+
+  @media (--viewport-max-mobile-landscape) {
+    width: 100%;
+    float: none;
+  }
+}
+
+.skin-membersB {
+  background-color: #fff;
```
Comment (SiAdcock) [inline public/components/membership/_membership_b.css]: Find and Replace: `#fff;` > `#ffffff;`

**Comment by SiAdcock** [comment]
Looks good, dude! 👍

**Comment by svillafe** [comment]
It is working on prod.
screencapture-prod
